lib/takeaway.rb

```diff
@@ -0,0 +1,9 @@
+class Takeaway
+
+  attr_reader :menu
+
+  def initialize 
+    @menu = { "pizza" => "£4.50", "chips" => "£2", 
```

Comment:
Maybe have each menu item on its own line. 
pizza, 
chips,
kebab
It may become difficult to read when the menu becomes longer